Projects/hGraphQL/GraphQL/Contracts.graphql

```diff
@@ -4,6 +4,7 @@ query Contracts($locale: Locale!) {
     displayName
     switchedFromInsuranceProvider
     gradientOption
+    typeOfContract
```

Comment:
Would be great if we wouldn't have to fetch this to keep the app as generic as possible